### DISCUSSION TITLE
Disable the map but stay highlighted

### DISCUSSION THREAD

**Discussion by salmanedhi** [img-mapper/img-mapper#26] [Q&A] [answered]
Hi, is it possible to disable further clicking on the image but the parts which were highlighted should stay highlighted.  Like a read-only view where once the user submits he can only see the ones he highlighted but cannot select more..

Right now when I set the disabled property to true it cleans the whole image

**Comment by salmanedhi** [comment, marked as answer]
I figured it out myself, by setting the disabled property on each zone individually. I hope that is the right way

**Reply by NishargShah** [reply]
You can add disabled prop in the `ImageMapper` component and add `preFillColor` property into JSON and it will give you the same result
